devel/automake/Makefile

```diff
@@ -9,11 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=automake
 PKG_VERSION:=1.15
-PKG_RELEASE:=3
+PKG_RELEASE:=4
```

Comment:
Guess there is no need to change `PKG_RELEASE` here as there should be no change in package content.